CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+* Fixed bug in `normal_polygon` in `compas.geometry._core`.
```

Comment:
Since the changelog is user-facing, the references should be to the public API, not the internal implementation structure:
```suggestion
* Fixed bug in `normal_polygon` in `compas.geometry`.
```